client/src/pages/TermsAndConditions/TermsAndConditions.tsx

```diff
@@ -0,0 +1,75 @@
+import { Icon } from "semantic-ui-react";
```

Comment:
i don't know if i would do this as a page and not as a popup.
Look that when you click it in the signup then it takes you to another page and then what if you want to go back? there is no way.
And what if you filled the signup and then you want to continue to fill the form but you clicked on terms and now it's all gone. That's why popup is much better

Reply:
I see what you're saying, however doing this will take time and i think it's best that we open a card in Trello for that and ill get to that some time this week